config/schema/ultimate_cron.schema.yml

```diff
@@ -140,71 +139,80 @@ ultimate_cron.job.*:
       mapping:
         id:
           type: string
-          label: 'Scheduler'
+          label: 'Scheduler ID'
         configuration:
-          type: mapping
-          label: 'Scheduler configuration'
-          mapping:
-            rules:
-              type: sequence
-              label: Scheduler rules
-              sequence:
-                - type: string
-                  label: Scheduling rule
-            catch_up:
-              type: integer
-              label: 'Timeout (s) after job run'
+          type: ultimate_cron.plugin.scheduler.[%parent.id]
     launcher:
       type: mapping
       label: 'Scheduler'
       mapping:
         id:
           type: string
-          label: 'Launcher'
+          label: 'Launcher ID'
         configuration:
-          type: mapping
-          label: 'Scheduler configuration'
-          mapping:
-            timeouts:
-              type: mapping
-              label: Timeout settings
-              mapping:
-                lock_timeout:
-                  type: integer
-                  label: Lock timeout
-                max_execution_time:
-                  type: integer
-                  label: Max execution time
-            launcher:
-              type: mapping
-              label: Launcher settings
-              mapping:
-                max_threads:
-                  type: integer
-                  label: Max threads
-                thread:
-                  type: integer
-                  label: Thread
-                poorman_keepalive:
-                  type: boolean
-                  label: Keepalive
+          type: ultimate_cron.plugin.launcher.[%parent.id]
     logger:
       type: mapping
       label: 'Scheduler'
       mapping:
         id:
           type: string
-          label: 'Logger'
+          label: 'Logger ID'
         configuration:
-          type: mapping
-          label: 'Scheduler configuration'
-          mapping:
-            method:
-              type: string
-              label: Method
-            expire:
-              type: integer
-              label: Expiration
-            retain:
-              type: integer
-              label: Retain X amount of logs
+          type: ultimate_cron.plugin.logger.[%parent.id]
+
+ultimate_cron.plugin.scheduler.*:
+  type: mapping
+  label: 'Scheduler configuration'
+  mapping:
+    rules:
+      type: sequence
+      label: 'Scheduler rules'
+      sequence:
+        type: string
+        label: 'Scheduling rule'
+    catch_up:
+      type: integer
+      label: 'Timeout (s) after job run'
+
+ultimate_cron.plugin.launcher.*:
```

Comment:
Wildcard is for defaults. We have specific launcher, logger and scheduler plugins (just one launcher and logger, two scheduler). Each should have its own schema.

Additionally, we should be using the same schema for the global default settings, those should also be plugin specific (the UI is not yet but that's the next thing to change)